templates/dashboard/next.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+{% load staticfiles i18n %}
+{% load render_bundle from webpack_loader %}
+{% load version %}
+{% load serialize_messages from utils %}
+
+<html lang="{{ LANGUAGE_CODE }}" class="no-js">
+  <head>
+    <title>
+      {% block title %}
+        {% trans "Dashboard - Saleor" context "Dashboard default page title" %}
+      {% endblock %}
+    </title>
+    {% block meta %}
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
```

Comment:
I'm not sure if blocking site scaling is good idea

Reply:
That's our current template and we do want to disable scaling for responsive design and modal positioning to work properly.